repo/packages/H/history/0/config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "mesos/master/url": {
```

Comment:
I observed that as well, while looking through other examples. The property here is about the mesos http urls, not the zk urls. To not confuse the user with zk urls, I chose a different key. Any suggestions?